static/src/javascripts/projects/common/modules/navigation/newSticky.js

```diff
@@ -61,72 +61,80 @@ define([
                 };
 
                 var render = function (state) {
-                    return fastdom.read(function () {
```

Comment:
That's clean! What do you think of moving all the logic out of the promise and refactoring the write to essentially:

```
function render(state) {
  /* build headerCss, adBannerCss and scroll objects here */
  return fastdom.write(function () {
    $header.css(headerCss);
    $adBanner.css(adBannerCss);
    window.scrollTo(scroll.x, scroll.y);
  };
}
```